core/scripts/common/helpers.go

```diff
@@ -570,12 +583,20 @@ func CalculateLatestBlockHeader(env Environment, blockNumberInput int) (err erro
 	return err
 }
 
-// IsAvaxNetwork returns true if the given chain ID corresponds to an avalanche network or subnet.
+// IsAvaxNetwork returns true if the given chain ID corresponds to an avalanche network.
 func IsAvaxNetwork(chainID int64) bool {
 	return chainID == 43114 || // C-chain mainnet
-		chainID == 43113 || // Fuji testnet
-		chainID == 335 || // DFK testnet
-		chainID == 53935 // DFK mainnet
+		chainID == 43113 // Fuji testnet
+}
+
+// IsAvaxSubnet returns true if the given chain ID corresponds to an avalanche subnet.
+func IsAvaxSubnet(chainID int64) bool {
+	return chainID == 335 || // DFK testnet
```

Comment:
I suppose we never ran backwards mode on DFK or we would have run into this issue hehe

Reply:
yeah i think we missed it oops